src/rustAnalyzerConfig.ts

```diff
@@ -0,0 +1,65 @@
+import { FlatpakManifest } from "./terminal"
```

Comment:
I would also move the file into a new directory called integration or something like that and name the file `rustAnalyzer` because I still hope to do the same integration for other languages in the future